src/main/java/com/mrbysco/angrymobs/tweaks/LeapAtTargetTweak.java

```diff
@@ -49,11 +50,13 @@ public LeapAtTargetTweak(EntityType<? extends PathfinderMob> entity, int priorit
 	@Override
 	public void adjust(Entity entity, String id) {
 		if (entity instanceof Mob mob) {
-			mob.goalSelector.availableGoals.forEach(goal -> {
-				if (goal.getGoal() instanceof LeapAtTargetGoal) {
-					AngryMobs.LOGGER.info("Overriding existing AI goal for entity {} using tweak ID {}", entity(), id);
-				}
-			});
+			if (AngryConfig.COMMON.enableInfoLog.getAsBoolean()){
+				mob.goalSelector.availableGoals.forEach(goal -> {
+					if (goal.getGoal() instanceof LeapAtTargetGoal) {
+						AngryMobs.LOGGER.info("Overriding existing AI goal for entity {} using tweak ID {}", entity(), id);
+					}
+				});
+			}
```

Comment:
In other classes you put the if check inside the forEach?

Reply:
Could you change the other patched tweaks to also surround the forEach instead of having those be inside? As this makes the code inconsistent